### ISSUE TITLE
Will this be added to Stremio 5.0?

### ISSUE THREAD

**Issue by Mushishin** [Loukious/StremioDiscordRichPresence#4]

**Comment by Loukious** [comment]
Definitely, yes. I'll just wait for a week or something if they decide to drop a hotfix and then work on that one. Because the new version has some issues (HDR for example) and devs are working on fixes.

**Comment by Loukious** [comment]
Initial support for Stremio 5.0.0 has been added. Get the new version here https://github.com/Loukious/StremioDiscordRichPresence/releases/tag/v0.0.5
